datachannel.go

```diff
@@ -275,6 +305,10 @@ func (d *DataChannel) handleOpen(dc *datachannel.DataChannel, isRemote, isAlread
 	d.mu.Unlock()
 	d.setReadyState(DataChannelStateOpen)
 
+	// fire the dial handler immediately after this peer has opened
+	// the data channel (DCEP OPEN has been sent)
+	d.onDial()
```

Comment:
Going to move this to after `datachannel.Dial` in `open` above; leaving it here would result in it getting called by the remote peer as well, which isn't actually dialing (its sending an ACK back in response to the dialing side's OPEN)